socorro/processor/mozilla_transform_rules.py

```diff
@@ -1092,7 +1092,8 @@ def _predicate(self, raw_crash, raw_dumps, processed_crash, proc_meta):
         '''addons is a list of tuples containing (extension, version)'''
         addons = processed_crash.get('addons', [])
 
-        for extension, version in addons:
+        for addon in addons:
+            extension, version = addon.split(':', 1)
```

Comment:
Actually, if you're not going to use `version` replace it with a `_`. 
An even safer thing would be:
```python
extension = addon.split(':')[0]
```
However, if you do *this*, there's a chance that the `_predicate()` might return True on an `addon` string that doesn't actually have a `:` in it. Thus you can get the same, above mentioned, `ValueError` later in the `_action` method.